cmd/ocm-backplane/console/console.go

```diff
@@ -249,6 +247,11 @@ func (o *consoleOptions) run(cmd *cobra.Command, argv []string) error {
 	if err != nil {
 		return err
 	}
+	// If this is a test scenario, we throw an error as opposed of keeping this function in a infinite loop
```

Comment:
Why do we need to indicate the tests in the code here?

Reply:
This is due to this function which gets called by cleanUp()


cmd/ocm-backplane/console.go
```
func execActionOnTermination(action postTerminationAction) error {
	sigs := make(chan os.Signal, 1)
	// syscall SIGUSR1 is used for testing
	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
	done := make(chan bool, 1)
	sig := <-sigs
	fmt.Println(sig)
	done <- true
	err := action()
	if err != nil {
		return err
	}
	return nil
}
```

Signal.Notify creates a busy wait which blocks the execution of the code. And if I send a CTRL+C via go-routines it will fail the test case. 

The indication of test stops the execution prior to calling clean up to prevent the tests to be stuck.

There are some possible alternative solutions, I thought about them but I don't think any of them are great so I choose the easiest which is what I did above:

1. Use SIGUSR1 as a signal to terminate the executable - Use of Signals is generally discouraged since we can't guarantee that SIGUSR1 is implemented on all platforms.
2. Use interfaces to create a mock function - Significant changes to the code required.